docs/swagger.yaml

```diff
@@ -485,6 +485,13 @@ paths:
       operationId: updateArticle
       summary: Edit article
       description: Edit article with an article id
+      parameters:
+      - in : path
+        name : articleId
+        schema:
+          type: integer
+        required: true
+        description : Numeric ID of the article to get
       requestBody:
         description: Edit a new article by article userId
```

Comment:
@beblicarl I think you made a minor mistake here. It's supposed to be article Id right?